jujugui/static/gui/src/app/assets/javascripts/jujulib/terms.js

```diff
@@ -79,10 +79,61 @@ var module = module;
     },
 
     /**
+      Parse and format a term response into a friendly format.
+
+      @private _formatTerm
+      @param term {Object} A term response.
+    */
+    _formatTerm: function(term) {
+      var milliseconds = Date.parse(term['created-on']);
+      return {
+        user: term.user,
+        term: term.term,
+        revision: term.revision,
+        createdAt: new Date(milliseconds)
+      };
+    },
+
+    /**
+      Creates a record of the authenticated user's agreement to a revision of
+      a Terms and Conditions document.
+
+      @public addAgreement
+      @param terms {Array} A list of terms to agree to. Each term should be an
+        object with the following parameters:
+        - name {String} The terms name.
+        - revision {Int} The terms revision.
+      @param callback {Function} A callback to handle errors or accept the
+        data from the request. Must accept an error message or null as its
+        first parameter and an authorization object as its second.
+    */
+    addAgreement: function(terms, callback) {
+      var self = this;
+      var handler = function(error, response) {
+        if (error !== null) {
+          callback(error, null);
+          return;
+        }
+        var terms = response.agreements.map(function(term) {
+          return self._formatTerm(term);
+        });
+        callback(null, terms);
+      }
+      var url = this.url + '/agreement';
+      var payload = terms.map(function(term) {
+        return {
+          termname: term.name,
+          termrevision: term.revision
```

Comment:
According to the code they're just lowercase, one word: https://github.com/CanonicalLtd/terms-service/blob/master/params/params.go#L59